userland/userland_generic.ld

```diff
@@ -20,49 +20,10 @@ MEMORY {
 }
 
 SECTIONS {
-    /* Text section, Code! */
+    /* Text section, Code! This needs to be first. */
     .text :
     {
         _text = .;
```

Comment:
I'm opposed to moving this _out_ of the control of the application. I think putting this in elf2tab unnecessarily ossifies this code.

In particular, we ought to move towards giving the app _more_ flexibility in how their flash and ram is laid out (currently the specific order and spacing of sections is fixed by elf2tab in a way that it really shouldn't) and moving this code out of the linker script would ossify that even more.